course05/homework01/skeleton_twyleg/string.h

```diff
@@ -0,0 +1,41 @@
+// Some copyright 2020
+
+#pragma once
+
+#include <string>
+#include <ostream>
+#include <array>
+#include <cstring>
+
+namespace cppcourse {
+
+class String
+{
+public:
+	String();
+    String(std::string);
```

Comment:
This will copy std::string which causes extra unnecessary work (and allocates), you should pass it as const ref

Reply:
- [ ] const ref